orm/src/test/java/org/hibernate/search/test/spatial/SpatialIndexingTest.java

```diff
@@ -180,6 +182,7 @@ public void testDistanceSort() throws Exception {
 
 	@Test
 	@TestForIssue(jiraKey = "HSEARCH-1708")
+	@Category(ElasticsearchSupportInProgress.class)
```

Comment:
I spent some time in LuceneHSQuery yesterday and I think I might as well fix it. Commit on the way.